draft-ar-pquip-pqc-engineers.md

```diff
@@ -231,9 +231,9 @@ These algorithms are not a drop-in replacement for classical asymmetric cryptogr
 The fourth-round of the NIST process focuses only on KEMs. The goal of that round is to select an althernative algorithm that is based on different hard problem than Kyber.
 The candidates still advancing for standardization are:
 
-* [Classic McEliece](https://classic.mceliece.org/): Based on the hardness of syndrome decoding of Goppa codes
-* [BIKE](https://bikesuite.org/): Based on the the hardness of syndrome decoding of QC-MDPC codes.
-* [HQC](http://pqc-hqc.org/) : Based on the hardness of syndrome decoding of Hamming Quasi-cyclic codes.
+* [Classic McEliece](https://classic.mceliece.org/): Based on the hardness of syndrome decoding of Goppa codes. Goppa codes are a class of error-correcting codes that can correct a certain number of errors in a transmitted message. The decoding problem involves recovering the original message from the received noisy codeword.
+* [BIKE](https://bikesuite.org/): Based on the the hardness of syndrome decoding of QC-MDPC codes. Quasi-Cyclic Moderate Density Parity Check (QC-MDPC) code are a class of error correcting codes that leverages bit flipping technique to efficiently correct the Hamming weight proportional to square root of the code length.
```

Comment:
I would re-phrase the last line of BIKE as follows: Quasi-Cyclic Moderate Density Parity Check (QC-MDPC) code are a class of error correcting codes that leverages bit flipping technique to efficiently correct errors." You can add more details after this line or stop.

Reply:
Sounds good.